fixtures/11c2f501cb0d488fad7eba3bab2a8c11

```diff
@@ -0,0 +1,10 @@
+ [
+   	{
+       	"title": "An inquiry into the wealth of nations",
+        "text": "This string seeks to help you understand the problem set"
+      },
+      {
+        "title": "From third world to first world",
+        "text": "This string is also to help you understand the problem set"
+      }
+    ]
```

Comment:
Why is your generated file pushed? I would suggested that you have a method in your code that clears your generated file from your fixtures folder.